services/kuber/templates/autoscaler-adapter.goyaml

```diff
@@ -3,9 +3,13 @@ apiVersion: v1
 kind: Service
 metadata:
   name: autoscaler-adapter-{{ .ClusterID }}
+  labels:
+    app.kubernetes.io/part-of: claudie
+    app.kubernetes.io/name: autoscaler-adapter-{{ .ClusterID }}
```

Comment:
By the way, if you wanted to utilize the templating with the ClusterID, then the `app.kubernetes.io/instance` label would be the appropriate one to use.
Once more, I refer to https://kubernetes.io/docs/concepts/overview/working-with-objects/common-labels/#labels.

So, just to be crystal clear about this for the **autoscaler resources**, I suggest precisely:
```
app.kubernetes.io/part-of: claudie
app.kubernetes.io/name: cluster-autoscaler
app.kubernetes.io/instance: cluster-autoscaler-{{ .ClusterID }}
```

Does this sound good to you @MiroslavRepka ?

Reply:
The `app.kubernetes.io/instance: cluster-autoscaler-{{ .ClusterID }}` has to be then changed in the Service selector if I am not mistaken (to point to the right pod).

Otherwise, I agree with your proposal @MarioUhrik

Reply:
Good one. I totally forgot about the `app.kubernetes.io/instance` label

Reply:
You're right, the service selector is a problem.
How about this then:

```
app.kubernetes.io/part-of: claudie
app.kubernetes.io/name: cluster-autoscaler
app.kubernetes.io/instance: cluster-autoscaler-{{ .ClusterID }}
```
and
```
app.kubernetes.io/part-of: claudie
app.kubernetes.io/name: cluster-autoscaler-adapter
app.kubernetes.io/instance: cluster-autoscaler-adapter-{{ .ClusterID }}
```

Reply:
@MiroslavRepka @Archisman-Mridha , while we're at it, we can also add these to both the adapter and the autoscaler, thus unifying them slightly

```
app.kubernetes.io/component: cluster-autoscaler`